lib/streamlit/elements/widgets/button.py

```diff
@@ -709,14 +709,14 @@ def _download_button(
         serde = ButtonSerde()
 
         button_state = register_widget(
-            "download_button",
             download_button_proto,
             on_change_handler=on_click,
             args=args,
             kwargs=kwargs,
             deserializer=serde.deserialize,
             serializer=serde.serialize,
             ctx=ctx,
+            value_type="trigger_value",
```

Comment:
nit: at some point, we maybe should rename this to `bool_trigger_value`

Reply:
Do you think it would make sense to do it now? Or is that a bigger refactoring?

Reply:
Oh, I think the problem is that it's also the name in the proto .. and we still have this "commitment" to not do changes. Maybe add a comment in the proto to rename it. And we should probably discuss about this commitment at offsite ..